x-pack/plugins/actions/server/lib/is_conector_deprecated.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { PreConfiguredAction, RawAction } from '../types';
+
+export type ConnectorWithOptionalDeprecation = Omit<PreConfiguredAction, 'isDeprecated'> &
+  Pick<Partial<PreConfiguredAction>, 'isDeprecated'>;
+
+export const isConnectorDeprecated = (
+  connector: RawAction | ConnectorWithOptionalDeprecation
+): boolean => {
+  if (connector.actionTypeId === '.servicenow' || connector.actionTypeId === '.servicenow-sir') {
```

Comment:
Do we not need to worry about `.servicenow-itom`?

Reply:
Christos is on PTO, so I'll leave that enhancement to another time (seeing  as this is the existing functionality on `main`).

Thanks for the feedback @ymao1

Reply:
Thank you @gmmorris for this PR 🚀 ! `.servicenow-itom` does not use an application so we do not have to worry about it.